app/api/firebase/FirebaseAuthManager.ts

```
@@ -84,6 +85,41 @@ export class FirebaseAuthManager extends FirebaseManager {
   }
+  
+  /**
+   * Sign in anonymously with a specific UUID
+   * @param uuid User UUID to use for anonymous authentication
+   * @param displayName Optional display name for the user
+   */
+  public async signInAnonymously(uuid?: string, displayName?: string): Promise<UserInfo> {
+    if (!this.getApp()) throw new Error('Not connected to Firebase');
+
+    try {
+      const auth = getAuth(this.getApp()!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)